src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -1,6 +1,31 @@
 <template>
 	<system-page>
 		<div class="page-content" style="max-width: 20rem;">
+			<div v-if="passwordless">
+				<div class="tw-flex tw-justify-center tw-items-center tw-relative tw-mb-2">
+					<div
+						v-if="fetchedLogoUrl"
```

Comment:
Moved this up. So now, along with the guards for partnerContentId, the Kiva logo shows up for passwordless even if there is no partner associated with the flow.